packages/blade/src/components/ActionList/ActionListNoResults.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box } from '~components/Box';
+import { SearchIcon } from '~components/Icons';
+import { Text } from '~components/Typography';
+
+const ActionListNoResults = (): React.ReactElement => {
+  return (
+    <Box textAlign="center" padding="spacing.5">
+      <SearchIcon size="xlarge" color="surface.text.normal.lowContrast" />
+      <Text marginTop="spacing.5" weight="bold">
+        No Search Result Found
+      </Text>
+      <Text marginTop="spacing.3" size="small" type="subtle">
+        Try Searching for a Different Text
```

Comment:
Is this on design? Can we not let our consumers also decide what to show here?

Reply:
With Controlled API we do allow that. Can think of exposing additional props if its something that people want to override very often.


On design we have this but it will be only possible with controlled API since it has too much of project-specific info- 

<img width="663" alt="image" src="https://github.com/razorpay/blade/assets/30949385/166ca530-e6ce-4aa1-a0f8-5cd7d167adf8">

Maybe I should not title case it and use `Try searching for a different value`. Will check this with designers at the end. 🤔

Reply:
So if I want to simply change the text, I'll have to change my logic and use controlled API instead. I suspect this use-case might come up soon and we might have to let users customise this text. Anyway, makes sense to defer that decision until the use-case comes up